### PR TITLE
HAWQ-1491 - create usage docs for HiveVectorizedORC profile

### PR DESCRIPTION
**Pull request by lisakowen**
update hawq docs for new HiveVectorizedORC profile.
- add example to hive plug-in page
- include the profile and accessor/fragmenter/resolver classes in the appropriate tables in the other docs

### CODE REVIEW

File: markdown/pxf/HivePXF.html.md.erb [hunk 1]
```diff
@@ -495,9 +500,16 @@ Use the `HiveORC` profile to access ORC format data. The `HiveORC` profile provi
...
     - `IN` operator on arrays of `int2`, `int4`, `int8`, `boolean`, and `text`
 
-- Complex type support - You can access Hive tables composed of array, map, struct, and union data types. PXF serializes each of these complex types to `text`.
+When choosing an ORC-supporting profile, consider the following:
+
+- The `HiveORC` profile supports complex types. You can access Hive tables composed of array, map, struct, and union data types. PXF serializes each of these complex types to `text`.  
+
+    The `HiveVectorizedORC` profile does not support complex types.
+
+- The `HiveVectorizedORC` profile reads 1024 rows of data, while the `HiveORC` profile reads only a single row at a time.
```
Comment (sansanichfb) [inline markdown/pxf/HivePXF.html.md.erb]: profile reads 1024 rows of data at once

File: markdown/pxf/HivePXF.html.md.erb [hunk 2]
```diff
@@ -565,6 +577,44 @@ In the following example, you will create a Hive table stored in ORC format and
...
+
+**Note**: The `HiveVectorizedORC` profile does not support the timestamp data type and complex types.
+
+1. Start the `psql` subsystem:
+
+    ``` shell
+    $ psql -d postgres
+    ```
+
+2. Use the PXF `HiveVectorizedORC` profile to create a queryable HAWQ external table from the Hive table named `sales_info_ORC` that you created in Step 1 of the previous example. The `FORMAT` clause must specify `'CUSTOM'`. The `HiveVectorizedORC` `CUSTOM` format supports only the built-in `'pxfwritable_import'` `formatter`.
```
Comment (sansanichfb) [inline markdown/pxf/HivePXF.html.md.erb]: queryable - maybe readable?

File: markdown/pxf/ReadWritePXF.html.md.erb [hunk 3]
```diff
@@ -105,6 +105,18 @@ Note: The <code class="ph codeph">DELIMITER</code> parameter is mandatory.
 <li>org.apache.hawq.pxf.service.io.GPDBWritable</li>
 </ul></td>
 </tr>
+<tr class="even">
+<td>HiveVectorizedORC</td>
+<td>Optimized block read of a Hive table where each partition is stored as an ORC file.
```
Comment (sansanichfb) [inline markdown/pxf/ReadWritePXF.html.md.erb]: People might get confused with HDFS block, so we can maybe use bulk/batch read.

**Comment by lisakowen** [comment]
thanks for the view, @sansanichfb.  i have incorporated your requested edits.

File: markdown/pxf/HivePXF.html.md.erb [hunk 4]
```diff
@@ -565,6 +577,44 @@ In the following example, you will create a Hive table stored in ORC format and
     Time: 425.416 ms
     ```
 
+### <a id="using-hiveorc-profile"></a>Example: Using the HiveVectorizedORC Profile
+
+In the following example, you will use the `HiveVectorizedORC` profile to query the `sales_info_ORC` Hive table you created in the previous example.
+
+**Note**: The `HiveVectorizedORC` profile does not support the timestamp data type and complex types.
```
Comment (dyozie) [inline markdown/pxf/HivePXF.html.md.erb]: Just to avoid any potential confusion, let's change this to "**or** complext types."

**Comment by lisakowen** [comment]
thanks for reviewing, @dyozie - i have made your requested change.

**Pull request closed without merging**